### PR TITLE
Add Personal token to developer

### PR DESCRIPTION
**Pull request by flickz**
Resolves https://github.com/opencollective/opencollective/issues/5736
Dependent on https://github.com/opencollective/opencollective-api/pull/8315

https://user-images.githubusercontent.com/15707013/208422061-3c901c47-46cc-4372-8223-2182e3e578be.mov

### CODE REVIEW

**Comment by znarf** [comment]
What are we doing with the old "Applications" page.

Maybe we can keep it and add a link to the new Personal Token one.

**Comment by flickz** [comment]
> Maybe we can keep it and add a link to the new Personal Token one.

@znarf Just to understand, having "Application" on the profile menu point to "Settings / For Developer" sections right?

File: components/personal-token/PersonalTokensList.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            {showLoadingState
+              ? Array.from({ length: variables.limit }, (_, index) => <LoadingPlaceholder key={index} height="64px" />)
+              : data.individual.personalTokens.nodes.map(token => (
+                  <Flex key={token.id} data-cy="personal-token" alignItems="center">
+                    <Box mr={24}>
+                      <Avatar radius={64} collective={data.individual} />
+                    </Box>
+                    <Flex flexDirection="column">
+                      <P fontSize="18px" lineHeight="26px" fontWeight="500" color="black.900">
+                        {token.name ?? 'Unnamed token'}
```
Comment (Betree) [inline components/personal-token/PersonalTokensList.js:122]: I18n

**Review by Betree — COMMENTED** [review]
The code looks good! I still need to test it with latest API and we'll be good to go

**Comment by znarf** [comment]
> Just to understand, having "Application" on the profile menu point to "Settings / For Developer" section right?

No, keep the "Applications" page (empty) and add a link to the new section there. It's to do a smooth transition and avoid confusing people that are used to this Applications page.

**Comment by Betree** [comment]
I get an error when I try to create a personal token without expiration date 

image

**Review by Betree — APPROVED** [review]
Except the error reported above everything looks good to me!

**Review by Betree — COMMENTED** [review]
Except the error reported above everything looks good to me!

**Comment by Betree** [comment]
@flickz Let me know if you don't have the capacity to iterate on this one at this stage, I can help moving it forward

**Comment by flickz** [comment]
@Betree It is fixed now. Thanks

**Review by Betree — APPROVED** [review]
Looks good to me, will do a final QA round on staging

**Pull request merged**